src/features/conversations/components/ConversationPanel.module.css

```
@@ -87,6 +89,25 @@
+  background: #303438ad;
+  border-radius: 0.5rem;
+}
+
+.systemMessageText {
+  font-family: monospace;
+  font-size: 1rem;
+  font-weight: 100;
+  letter-spacing: 0.05em;
+  text-align: center;
+
+  color: rgb(120, 218, 254);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)